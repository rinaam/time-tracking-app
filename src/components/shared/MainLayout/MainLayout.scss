@import '../../../styles/spacing.scss';

.container {
    margin: 0 auto;
    padding: $spacing-s;
    max-width: 1200px;
}