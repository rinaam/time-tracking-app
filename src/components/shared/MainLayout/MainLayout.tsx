import React from 'react';
import './MainLayout.scss';

interface MainLayoutProps {
  children: React.ReactNode
}

const MainLayout:React.FC<MainLayoutProps> = ({ children }) => (
  <div className="container">{children}</div>
);

export default MainLayout;
