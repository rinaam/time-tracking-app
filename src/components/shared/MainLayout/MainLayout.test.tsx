import React from 'react';
import { render } from '@testing-library/react';
import MainLayout from './MainLayout';

it('should match snapshoot', () => {
  const wrapper = render(<MainLayout>Hello World</MainLayout>);
  expect(wrapper.asFragment()).toMatchSnapshot();
});
