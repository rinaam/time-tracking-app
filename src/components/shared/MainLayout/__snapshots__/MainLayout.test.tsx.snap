// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match snapshoot 1`] = `
<DocumentFragment>
  <div
    class="container"
  >
    Hello World
  </div>
</DocumentFragment>
`;
